creator-node/src/utils.js

```diff
@@ -132,7 +132,7 @@ async function rehydrateIpfsFromFsIfNecessary (req, multihash, storagePath, file
     req.logger.info(`rehydrateIpfsFromFsIfNecessary - Re-adding dir ${multihash}, stg path: ${storagePath}, filename: ${filename}, ipfsPath: ${ipfsPath}`)
     let findOriginalFileQuery = await models.File.findAll({
       where: {
-        storagePath: { [models.Sequelize.Op.like]: `%${multihash}%` },
+        dirMultihash: multihash,
```

Comment:
is this the big perf improvement here? can discuss separately, but per your slack comment with an index on the storagePath column there's not much perf diff between these two versions?